docs/LUA.md

```diff
@@ -5,7 +5,7 @@
 The mLRS Lua script provides a convenient way to change the Tx module and receiver settings.
 
 The Lua script works on both OpenTx and EdgeTx radios but there are two different versions depending on the display of your radio:
-1. If your radio has a 480x272 color screen (e.g. Jumper T16, Radiomaster TX16S) then use the "mLRS.lua" file
+1. If your radio has a 480x320 or 480x272 color screen (e.g. Jumper T15, Radiomaster TX16S) then use the "mLRS.lua" file
```

Comment:
I would like to suggest

1. If your radio has a 480x272 (e.g. Jumper T16, Radiomaster TX16S) or 480x320 (e.g. Jumper T15) color screen then use the "mLRS.lua" file

480x272 is more widely around than 480x320